### PR TITLE
Updated commit fix

### PR DESCRIPTION
**Pull request by rexdivakar**
Related to #1243 

Description:

Check list:
* [ ] New tests are added (if a new feature is added)
* [x] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

File: .github/workflows/code-style.yml [hunk 1]
```diff
@@ -14,10 +14,10 @@ jobs:
         autopep8 --recursive --in-place --aggressive --aggressive .
         black .
     - name: Commit and push changes
-      uses: stefanzweifel/git-auto-commit-action@v2.0.0
+      uses: stefanzweifel/git-auto-commit-action@v4
       with:
         commit_author_email: ''
```
Comment (vfdev-5) [inline .github/workflows/code-style.yml:19]: Please remove commit_author_email and commit_author_name

Comment (rexdivakar) [reply .github/workflows/code-style.yml:19]: ok

Comment (rexdivakar) [reply .github/workflows/code-style.yml]: Updated the changes ! please verify now @vfdev-5

**Comment by rexdivakar** [comment]
Can u please validate the changes and approve my PR @vfdev-5

**Comment by rexdivakar** [comment]
@vfdev-5 Hello
sorry to interrupt you
But as you know Hactobesfest rule is change
in Repo: Hactobesfest label and
in PR: Hactobesfest-accepted label should be there then It will count

In my PR Hactobesfest-accepted are show if can Please can label it !!

**Comment by vfdev-5** [comment]
@rexdivakar please see here : https://github.com/pytorch/ignite/issues/1257#issuecomment-704188385

**Comment by rexdivakar** [comment]
Thanks for the clarity !! 👌

**Pull request merged**